api/src/main/resources/api/service_openapi.yaml

```diff
@@ -60,6 +111,34 @@ components:
         type: string
 
   responses:
+    GetEntityResponse:
+      description: Response to get an entities.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Entity'
+
+    GetUnderlayResponse:
+      description: Response to get an underlay.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Underlay'
+
+    ListEntitiesResponse:
+      description: Response to list the entities.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/EntitiesList'
```

Comment:
I feel obligated to link this: https://cloud.google.com/apis/design/design_patterns#list_pagination

However, I don't know that I would go through the trouble for these metadata APIs

Reply:
Oh, good point. Hmmm, yeah it seems a little overkill, but consistency is a good argument. Eh, I'll start with it in and if implementation time it doesn't seem worth it I'll rip it out again.
PTAL